src/applications/claims-status/containers/AppealsV2DetailPage.jsx

```diff
@@ -2,17 +2,16 @@ import React from 'react';
 import { useOutletContext } from 'react-router-dom-v5-compat';
 
 import Issues from '../components/appeals-v2/Issues';
-import { APPEAL_TYPES, addStatusToIssues } from '../utils/appeals-v2-helpers';
+import { getTypeName, addStatusToIssues } from '../utils/appeals-v2-helpers';
 
 export default function AppealsV2DetailPage() {
   const [appeal] = useOutletContext();
   const issues = addStatusToIssues(appeal.attributes.issues);
-  const isAppeal =
-    appeal.type === APPEAL_TYPES.appeal || appeal.type === APPEAL_TYPES.legacy;
+  const appealType = getTypeName(appeal);
 
   return (
     <div id="tabPanelv2detail">
-      <Issues issues={issues} isAppeal={isAppeal} />
+      <Issues issues={issues} appealType={appealType} />
```

Comment:
We need to pass the appeal type in so that we can filter by type in the `Issues` component.